src/pages/events/commands.md

```diff
@@ -119,12 +121,25 @@ You can also subscribe to a plugin event if it was registered in the `app/etc/co
 
 `--force`, `-f` Forces subscription to the event, even if it hasn't been defined locally.
 
+`--parent <event code>` Specifies the name of the event to use as the basis of an event rule. If you specify this option, you must also specify the `--rules` option.
+
+`--rules=<field-name>|<operator>|<value>` Defines a rule that will be applied to the subscribed event. You can apply multiple rules to an event, but each rule must be defined separately. A rule definition must specify the field to be evaluated, an operator, and the value to be evaluated, in that order. The field name in a rule definition must match a field specified with the `--fields` option.
```

Comment:
We currently don't enforce that the field name in the rule definition matches a field specified with the --fields option (a user could create a rule to check that a product price is greater than $20 but only have the name of the product being transmitted to App Builder, for example)